crypto/o_str.c

```diff
@@ -143,26 +143,24 @@ int OPENSSL_hexstr2buf_ex(unsigned char *buf, size_t buf_n, size_t *buflen,
 
     for (p = (const unsigned char *)str, q = buf, cnt = 0; *p; ) {
         ch = *p++;
-        if (ch == ':')
+        /* A seperator of CH_ZERO means there is no seperator */
```

Comment:
separator not seperator

Reply:
> Side note: I've been pondering refactoring all possible variants of hexdump output into a BIO

In that case it sounds like I should just make these functions internal for now.